proto/xudrpc.proto

```diff
@@ -416,6 +439,23 @@ message Balance {
   uint64 unconfirmed_wallet_balance = 6 [json_name = "unconfirmed_wallet_balance"];
 }
 
+message BalanceAlert {
+  // The total balance.
+  uint64 total_balance = 1 [json_name = "total_balance"];
```

Comment:
I think this would be better called `capacity` since that is the term I'm most used to seeing when referring to channel sizes (see for example https://1ml.com/channel/722305472288391168). And the comment can say "The total capacity of the channel this alert is for"

Reply:
But we show total trading balance, instead of channel balances?

Reply:
Maybe I was confused then, this field is the total balance that the node has across all channels? If so then it's definitely not capacity, but that would seem out of place to me.